tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -114,4 +114,127 @@ class WskWebActionsTests
             authorizedResponse.statusCode shouldBe 200
             authorizedResponse.body().asString() shouldBe namespace
     }
+
+    it should "ensure --web and --no-web flags set the proper annotations" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "webaction"
+            val file = Some(TestUtils.getTestActionFilename("echo.js"))
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) =>
+                    action.create(name, file, web = Some(true))
+            }
+
+            val stdout = wsk.action.get(name, fieldFilter = Some("annotations")).stdout
```

Comment:
think you can collapse the 3 repetitions with a Seq(true, false, true).foreach ...